.travis.yml

```diff
@@ -6,5 +6,5 @@ rvm:
   - '2.2.4'
   - '2.3.0'
 script:
-  - bundle exec rubocopter --commit origin/master
+  - gem install rubocop
```

Comment:
this appears to swap out *running* rubocopter for *installing* rubocop.  is this intended?